docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -154,4 +164,32 @@ Map<String, SecurityCheckResult> results = securityService.getCheckExecutor().ad
 
 // adding self defence checks and executing these checks asynchronously
 Map<String, Future<SecurityCheckResult> results = securityService.getAsyncCheckExecutor().addCheck(new customSecurityCheck()).addCheck(SecurityCheckType.<check_type>).addCheck(SecurityCheckType.<check_type>).execute();
-----
\ No newline at end of file
+----
+
+=== Reporting Self-defence checks results via the Metrics service
+
+In order to report the results of Self-defence utilize the link:./metrics[Metrics] service in conjunction with the Security service.
+
+Individual checks can be reported via the `checkAndSendMetric()` method:
+
+[source, java]
+----
+MetricsService metricsService = mobileCore.getInstance(MetricsService.class);
+SecurityService securityService = activity.mobileCore.getInstance(SecurityService.class);
+SecurityCheckResult result = securityService.checkAndSendMetric(SecurityCheckType.IS_ROOTED, metricsService);
+----
+
+Metrics information containing the results of the checks run via an Executor will be automatically sent if the Executor is supplied an instance of the `MetricsService`:
+
+[source, java]
+----
+MetricsService metricsService = mobileCore.getInstance(MetricsService.class);
+SyncSecurityCheckExecutor syncCheckExecutor = SecurityCheckExecutor.Builder.newSyncExecutor(this.getContext())
```

Comment:
Maybe a comment for `this.getContext()`?

Reply:
Getting a context is *very* central for native android development so I wouldn't say it needs an extra comment here.